gulpfile.js

```diff
@@ -198,7 +204,7 @@ gulp.task('build', gulp.series('clean', gulp.parallel('html', 'browserify', 'sas
 gulp.task('build-prod', gulp.series(function(callback) {
   production = true;
   callback();
-}, 'lint', 'build'));
+}, 'lint', 'build', 'copy-json'));
```

Comment:
I think this should be part of the `build` task, independent from the production flag.